lib/p2p/Pool.ts

```diff
@@ -245,11 +245,11 @@ class Pool extends EventEmitter {
    * If the node is banned, already connected, or has no listening addresses, then do nothing.
    * Additionally, if we're already trying to connect to a given node also do nothing.
    * @param nodes a collection of nodes with a `forEach` iterator to attempt to connect to
-   * @param ignoreKnown whether to ignore nodes we are already aware of, defaults to false
+   * @param allowKnown whether to allow nodes we are already aware of, defaults to true
```

Comment:
Could you make this a bit clearer by saying "whether to allow *connecting to* nodes we are already aware of"